### PR TITLE
CASE 1.2.0 Validation

### PR DESCRIPTION
**Pull request by kchason**
Re-adds CASE 1.0.0 validation if https://github.com/casework/CASE-Implementation-UFED-XML/pull/4 is merged temporarily.

If https://github.com/casework/CASE-Implementation-UFED-XML/pull/4 is unused, this pull request should be closed.

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
I suggest this PR be scope-crept to bump validation to CASE 1.1.0.  The `case-utils` version providing CASE 1.1.0 also incorporates a concept typo checker, and that functionality could become a significant component of reviewing CASE conformance.

**Comment by ajnelson-nist** [comment]
I've bumped CASE validation to using CASE 1.2.0.

If Continuous Integration passes, I think this can be merged.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## XML SAX parser for UFED/Cellebrite.
 
 [![Continuous Integration](https://github.com/casework/CASE-Implementation-UFED-XML/actions/workflows/cicd.yml/badge.svg)](https://github.com/casework/CASE-Implementation-UFED-XML/actions/workflows/cicd.yml)
+![CASE Version](https://img.shields.io/badge/CASE%20Version-1.1.0-green)
```
Comment (kchason) [inline README.md]: This should get bumped as well

Comment (ajnelson-nist) [reply README.md]: Roger.  Bumping...

Comment (ajnelson-nist) [reply README.md]: Bumped.  Thanks!

**Comment by kchason** [comment]
I can't approve this since I opened the initial PR, but I'm good with this merging with the 1.2.0 changes reflected.

**Pull request merged**